pubtools/_content_gateway/push_staged_cgw.py

```diff
@@ -0,0 +1,97 @@
+from pushsource import CGWPushItem
```

Comment:
The import grouping oesn't adhere to pep 8: https://peps.python.org/pep-0008/#:~:text=Imports%20are%20always%20put%20at,Related%20third%20party%20imports.

Reply:
Updated